containers/Ideas/IdeaCreate.js

```diff
@@ -1,23 +1,23 @@
-import React, {useState, useEffect} from 'react'
+import React, {useEffect,useState} from 'react'
 import { Alert, View } from 'react-native'
+import AsyncStorage from '@react-native-async-storage/async-storage'
 import { Formik } from 'formik'
 import * as yup from 'yup'
-import AsyncStorage from '@react-native-async-storage/async-storage'
+
+import API from '../../BaseApi'
 import { ButtonSubmit } from '../../components/ButtonSubmit'
 import { ButtonTextInput, ButtonTextInputFieldContainer } from '../../components/ButtonTextInput'
+import {
```

Comment:
more of a personal preference:

```
import {
  DropdownFormField,
  DropdownFormFieldContainer,
  TextInputFormField
} from '../../components..'
```

Reply:
agreed, I just used the auto-formatting of eslint here